var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -17,34 +17,34 @@ class PyPyside2(PythonPackage):
     # https://wiki.qt.io/Qt_for_Python_Development_Getting_Started
 
     version("develop", tag="dev")
+    version("5.15.10", tag="v5.15.10-lts-lgpl", submodules=True)
     version("5.15.2.1", tag="v5.15.2.1", submodules=True)
     version("5.14.2.1", tag="v5.14.2.1", submodules=True)
-    version("5.13.2", tag="v5.13.2", submodules=True)
-    version("5.13.1", tag="v5.13.1", submodules=True)
-    version("5.13.0", tag="v5.13.0", submodules=True)
-    version("5.12.5", tag="v5.12.5", submodules=True)
 
     variant(
         "doc",
         default=False,
         description="Enables the generation of html and man page documentation",
     )
 
-    depends_on("python@2.7.0:2.7,3.5.0:3.5,3.6.1:", type=("build", "run"))
-    depends_on("python@2.7.0:2.7,3.5.0:3.5,3.6.1:3.8", when="@:5.14", type=("build", "run"))
+    # dependency matrix from: https://wiki.qt.io/Qt_for_Python
+    depends_on("python@:3.10", when="@5.15.8:", type=("build", "run"))
+    depends_on("python@:3.9", when="@5.15.1:5.15.7", type=("build", "run"))
+    depends_on("python@:3.8", when="@:5.15.0", type=("build", "run"))
+    depends_on("python@:3.7", when="@:5.13", type=("build", "run"))
 
     depends_on("cmake@3.1:", type="build")
     # libclang versioning from sources/shiboken2/doc/gettingstarted.rst
-    depends_on("llvm@6", type="build", when="@5.12:5.13")
-    depends_on("llvm@10", type="build", when="@5.15")
+    depends_on("llvm@10+clang", type="build", when="@5.15")
+    depends_on("llvm@6+clang", type="build", when="@5.12:5.13")
     depends_on("py-setuptools", type="build")
     depends_on("py-packaging", type="build")
     depends_on("py-wheel", type="build")
     # https://bugreports.qt.io/browse/PYSIDE-1385
     depends_on("py-wheel@:0.34", when="@:5.14", type="build")
     # in newer pip versions --install-option does not exist
     depends_on("py-pip@:23.0", type="build")
-    depends_on("qt@5.11:+opengl")
+    depends_on("qt@5.12:+opengl")
```

Comment:
qt 4.6+ for 5.12.5 according to `docs/index.rst`. You'll need to check to see what the requirements are per version. It would also be helpful to add a comment saying where these come from.

Reply:
Could also be in `setup.py`, although that says 5.11+. Not sure which to trust